src/input.scss

```diff
@@ -10,16 +10,17 @@ $block: #{$fd-namespace}-input;
 .#{$block} {
   @include fd-form-text();
   @include fd-var-size("height", $fd-forms-height, --fd-forms-height);
+  @include fd-var-size("line-height", $fd-forms-height, --fd-forms-height);
```

Comment:
Since we are touching the component maybe we should migrate it to use the parameters from the theming package